### DISCUSSION TITLE
Malware?

### DISCUSSION THREAD

**Discussion by jensenbox** [mistweaverco/bananas#11] [Q&A] [answered]
I just downloaded to give it a whirl on my mac and got this:

<img width="365" alt="image" src="https://github.com/user-attachments/assets/9d43d86d-76c8-449d-b81b-539a050e2733">
 
I am on Sequoia 15.0.1 (M3)

**Comment by gorillamoe** [comment]
No malware, but not signed and notarized by Apple. I tried enrollment of the apple developer program today, so we can sign and notarize the app in the CI on build, but apple seems to have had a maintenance window when I tried.

You need to manually allow the access for now:

https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unknown-developer-mh40616/mac

**Reply by madeus** [reply]
> No malware, but not signed and notarized by Apple. I tried enrollment of the apple developer program today, so we can sign and notarize the app in the CI on build, but apple seems to have had a maintenance window when I tried.
> 
> You need to manually allow the access for now:
> 
> https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unknown-developer-mh40616/mac

You have to do exactly as described in the link. It works. ;-)

**Comment by jensenbox** [comment]
It will not open even when I click the "Open Anyway" button.

![image](https://github.com/user-attachments/assets/d2ff75dc-b3c1-4e32-b139-1eebbb6f0323)

**Comment by gorillamoe** [comment]
I'm no MacOS expert by any means, so maybe @madeus or @rumblemumble can chime in on how they managed to set an exception for Bananas to run on MacOS.

**Comment by gorillamoe** [comment]
Maybe related: https://github.com/orgs/Homebrew/discussions/3088

So the temporary fix would be:

```
codesign --force --deep --sign - /Applications/bananas.app
```

**Reply by gorillamoe** [reply, marked as answer]
Fixed in v0.0.12 and upwards
